lib/Github/Api/Deployment/Environments.php

```diff
@@ -1,13 +1,15 @@
 <?php
 
-namespace Github\Api;
+namespace Github\Api\Deployment;
+
+use Github\Api\AbstractApi;
 
 /**
  * Listing, creating and updating environments.
  *
  * @link https://docs.github.com/en/rest/deployments/environments?apiVersion=2022-11-28#
  */
-class Environment extends AbstractApi
+class Environments extends AbstractApi
```

Comment:
This is technically a BC break, but I think we can do it as it's new class since last release (today)

Reply:
Hi @acrobat I was thinking the same, becuase I've added first Environments but after realised that Environments are part of Deployment.